finetune/base.py

```diff
@@ -296,7 +305,12 @@ def _distribute_strategy(self, visible_gpus):
             resolved_gpus = all_gpus()
 
         resolved_gpus_string = ['/gpu:{}'.format(gpu) for gpu in resolved_gpus]
-        distribute_strategy = tf.distribute.experimental.CentralStorageStrategy(resolved_gpus_string or None)
+
+        if self.config.use_mirrored_distribution and len(resolved_gpus) > 1:
+            distribute_strategy = tf.distribute.MirroredStrategy()
+        else:
+            distribute_strategy = tf.distribute.experimental.CentralStorageStrategy(resolved_gpus_string or None)
```

Comment:
Until distribution strategy `per_process_gpu_memory_fraction` gets fixed we'll have to deal with that here.

Reply:
Should be caught in a follow-up rebase since 0.8.2 merge is in